webrtc.html

```diff
@@ -11087,16 +11087,10 @@ <h3>Introduction</h3>
         even past the associated peer connection being <a data-lt="close"
           href="#dom-rtcdatachannel-close"><code>close</code></a>d.
       </p>
-      <p>Only a few monitored objects have shorter lifetimes. For these objects,
```

Comment:
The link "#dom-rtcdatachannel-close" seems wrong, should probably be "dom-rtcpeerconnection-close".